packages/amplify-graphql-api-construct-tests/src/__tests__/utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import { isOperationAuthInputApiKey } from '../utils';
```

Comment:
I needed to verify some predicate type behavior, so I left this in place. It only runs during actual E2E builds, so it's not much use as a unit test, but it does verify that the predicate function is working as expected.